pkg/model/file.go

```diff
@@ -13,7 +13,71 @@
 
 package model
 
-import "path/filepath"
+import (
+	"os"
+	"path/filepath"
+)
+
+var (
```

Comment:
opinion: can we put this at a file alone? like file_names or something like that :/